ami_builder.py

```diff
@@ -105,5 +105,10 @@ def handler(event, context):
     json_content = json.loads(open('event.json', 'r').read())
     try:
         handler(json_content, None)
+    except Exception as e:
```

Comment:
Sorry Ian, this one's my fault.  Can you swap the order of the 2 `except` blocks please.  In this order the 2nd one is never going to fire because every exception will be caught by the first one as it's so generic.